src/components/Search/Search.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styles from './Search.module.scss'
+import LinkTo from 'components/util/LinkTo/LinkTo'
+
+const Search = () => {
+  return (
+    <div className={styles.search}>
+      <form className="usa-search usa-search--big" role="search">
+        <select className="usa-select usa-button" name="options" id="options">
```

Comment:
both form inputs should have a label element (that can be `.usa-sr-only`) in order to be accessible!

Reply:
THANKS!